system_test/aest_commands_SUITE.erl

```diff
@@ -63,7 +63,7 @@ epoch_commands(Cfg) ->
     ?assertEqual(ExpPeerKey, PeerKey),
 
     {0, Output3} = aest_nodes:run_cmd_in_node_dir(node1,
-        ["bin/epoch", "check_config", "epoch.yaml"], Cfg),
+        ["bin/epoch", "check_config", "../epoch.yaml"], Cfg),
```

Comment:
It looks like this was cherry-picked and included from master rather than rebasing on master. Ok anyway.

Reply:
Same as for `aec_id`...